README.md

```diff
@@ -0,0 +1,291 @@
+- Attractive Images & Design
+- Likes/Comments
+- Dynamic page (data is retrieved from JSON file)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://faisalakb.github.io/Javascript-capston/dist/)
+
+## 🚀 Live Recording <a name="live-demo"></a>
+
+[The recording](https://drive.google.com/file/d/1267JBuQlm1FZi-uKGBSUqWSBA882vg8o/view?usp=sharing)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git version 2.38.x
+- node.js version > 12.x
+- IDE
+- browser (chrome, firefox, edge, safari)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd Javascript-capston
+
+  git https://github.com/faisalakb/Javascript-capston
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Javascript-capston
+  node npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+run live server
+
+<!--
+```sh
+  rails server
+```
+-->
+
+### Run tests
+
+To run tests, run the following command:
+Run Github Actions Test
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx eslint .
+  npm test
+```
+
+### Deployment
+
+This project is deployed at github pages you can clone it here. [Please click to clone](coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors
+
+👤 **Faisal Mumtaz**
+
+- GitHub: [@faisalakb](https://github.com/faisalakb)
+- Twitter: [@faisalakb786](https://twitter.com/Faisalakb786)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/faisal-mumtaz-514a221a6/)
+
+
+👤 **Essohanam TAMBANA**
+
+- GitHub: [@GhostEsso](https://github.com/GhostEsso)
+- Twitter: [@TambanaEssohanam](https://twitter.com/TambanaEssohana)
+- LinkedIn: [@TambanaEssohanam](https://www.linkedin.com/in/essohanam-tambana-62aa081a1/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Dynamic content.**
+- [ ] **Add functionality of Javascript API**
+- [ ] **Will add some background and beautification**
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/faisalakb/Javascript-capston/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star to this repositiory.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse full-stack curriculum for the inspiration and guidance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is under [MIT](https://github.com/faisalakb/Javascript-capston/blob/development/LICENSE) licensed.
```

Comment:
Please fix the broken LICENSE link, let's avoid broken links in our readme for a professional repository 🛠 👍 

![image](https://github.com/faisalakb/Javascript-capston/assets/78251927/d4d1b5cc-ef6a-4a2d-bf86-66ada1c4dce3)

Reply:
Resolve all the comments mention above